### PR TITLE
Add FromStr impl to Configs #189

### PR DESCRIPTION
**Pull request by picklenerd**
Remove generation of config files from tests

### CODE REVIEW

**Comment by picklenerd** [comment]
todo: Make an issue and add it to the commit

File: dams-tests/tests/common.rs [hunk 1]
```diff
@@ -43,6 +40,11 @@ impl Party {
 
 #[allow(unused)]
```
Comment (marsella) [inline dams-tests/tests/common.rs:41]: This is not on you but we can probably take out this flag now :)

**Review by marsella — APPROVED** [review]
This looks good to me. I thought you'd have to take out the `#[non_exhaustive]` tag on the `Config`s, but maybe that's only to prevent manual construction of a Config, rather than construction from a string?

File: dams-tests/tests/common.rs (hunk 1 above)
Comment (picklenerd) [reply dams-tests/tests/common.rs]: `db` isn't used so it still returned an error but I changed it to `_db` so we could avoid the the `allow`. I got rid of the other unnecessary `#[allow(unused)]` blocks too.

**Pull request merged**